syntaxes/vscode-scl.tmLanguage.json

```diff
@@ -327,16 +367,14 @@
 		},
 		"strings": {
 			"name": "string.quoted.single.scl",
-			"begin": "'",
-			"end": "'",
 			"patterns": [
 				{
 					"name": "constant.character.escape.scl",
 					"match": "\\\\."
 				},
 				{
 					"name": "constant.character.escape.scl",
-					"match": "\\$."
+					"match": "'[^'\\$]*(?:\\$[\\s\\S][^'\\$]*)*'"
```

Comment:
see #21

Reply:
My mistake, I did some testing and it got accidentally included in an unrelated commit.